core/chains/evm/client/node.go

```diff
@@ -61,6 +61,10 @@ var (
 		Name: "evm_pool_rpc_node_calls_success",
 		Help: "The approximate total number of successful RPC calls for the given RPC node",
 	}, []string{"evmChainID", "nodeName"})
+	promEVMPoolSendTransactionTiming = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Name: "evm_pool_rpc_node_send_tx_timing",
```

Comment:
Maybe should align with:
`keeper_check_upkeep_execution_time`
`pipeline_task_execution_time`
`pipeline_task_eth_call_execution_time`
etc.

Also we have `uptime_seconds` :shrug:

Reply:
`*_execution_time` sounds good to me, will switch to that.